docs/SUBWORKFLOWS.md

```diff
@@ -0,0 +1,128 @@
+
+#### Fill the `main:` section.
+
+Compose your workflow using the different modules and workflows you've included above.
+Modules using input channels just need to specify the appropriate channel :
+
+```
+<MODULE1> (channel_data1)
+```
+
+To connect two modules together you need to create a channel which takes one of the outputs of the first module. To do this, use the `.out` which allows you to select the outputs of the first module. Finally, specify which of the module's outputs you want :
+
+```
+channel_module2 = <MODULE1>.out.<output>
+<MODULE2> (channel_module2)
+```
+
+When a module requires multiple inputs, don't create several channels. Create one that contains the different inputs required by the modules. For this, you need an operator. One of the most useful is `.join`. Attention the order is important and must correspond to the desired order in the module :
+
+```
+channel_module3 = <MODULE2>.out.<output>.join(channel_data1).join(channel_data2)
+<MODULE3> (channel_module3)
+```
+
+> [!NOTE]
+> There are different types of operator depending on your needs. For a complete list, please refer to the nextflow documentation: : https://www.nextflow.io/docs/latest/operator.html#
+
+> [!WARNING]
+> The same applies to workflows, you can link modules to workflows and vice versa.
+
+At the same time, you need to edit the version files of our various modules. The first thing to do in the main is to create an empty channel:
+
+```
+ch_versions = Channel.empty()
+```
+
+Then, at the end of each module, incorporate the module version into the channel version.
+
+```
+channel_module2 = <MODULE1>.out.<output>
+<MODULE2> (channel_module2)
+ch_versions = ch_versions.mix(<MODULE2>.out.versions.first())
+```
+
+#### define your Workflow outputs.
+
+Once the `main` finish you can define the output that you want from the different modules or workflows.
+A workflow can declare one or more output channels using the `emit` keyword.
+
+```
+emit:
+    output1 = <MODULE1>.out.<output> // channel: [ val(meta), [ output ] ]
+    output2 = <MODULE2>.out.<output> // channel: [ val(meta), [ output ] ]
+```
+
+> [!NOTE]
+> As with `main`, you can create outputs containing several files using operators.
+
+Don't forget to also define the output for the version file :
+
+```
+    versions = ch_versions // channel: [ versions.yml ]
+```
+
+### Editing `./subworkflows/nf-scil/<name_of_your_workflow>/meta.yml` :
+
+Fill the sections you find relevant. There is a lot of metadata in this file, but you
+don't need to specify them all. At least define 3 `keywords`, describe the workflow'
+`inputs` and `outputs` in the order in which they appear, and add a `complete description` for the use of the subworkflow.
+
+### Editing `./test/subworkflows/nf-scil/<name_of_your_workflow>/main.nf` :
+
+### Editing `./test/subworkflows/nf-scil/<name_of_your_workflow>/nextflow.config` :
+
+## Run the tests to generate the test metadata file
+
+## Submit your PR
```

Comment:
I cannot suggest the change, since there is code snippet in it, but you can put this block at the end. I adapted from what I'm modifying in the MODULE.md file right now. Feel free to change to your liking !

## Lint your code

Run `prettier` on your new module, through the _nf-core_ command line :

```bash
nf-core subworkflows \
  --git-remote <your repository> \
  --branch <your branch unless main branch> \
  lint <subworkflow>
```

and fix all `errors` and as much as the `warnings`as possible. Refer to
[this section](./VALIDATION.md#code-standards-and-formatting) for further information.

## Submit your PR

Open a PR to the `nf-scil` repository master. We'll test everything, make sure it's
working and that code follows standards.

Once LGTM has been declared, wave to the maintainers and look at your hard work paying off.

PR merged !

Reply:
Can you send me the address to which the link in "this section" leads as I'm not sure of the direction.

Reply:
Oh my bad, It links to a new documentation section I'm working on, didn't thought about that. I'll get back to you in the day on that.